ci/code_checks.sh

```diff
@@ -309,6 +309,10 @@ if [[ -z "$CHECK" || "$CHECK" == "doctests" ]]; then
     pytest -q --doctest-modules pandas/core/arrays/boolean.py
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
+    MSG='Doctests dtypes/inference.py' ; echo $MSG
+    pytest -q --doctest-modules pandas/core/dtypes/inference.py -k "is_iterator"
```

Comment:
I don't think need to special case these - is doctest maybe just not being run?

Reply:
Doctests are being run, but only for specific files (and for some only for specific functions/methods).

https://github.com/pandas-dev/pandas/blob/2170f4e49bdd6e7bb0441f142fce9528c0c6c422/ci/code_checks.sh#L261-#L310

(Most recent version at the time of writing this)

Those are the only one who are currently being tested.